lms/djangoapps/instructor/access.py

```diff
@@ -45,7 +49,7 @@ def allow_access(course, user, level):
     """
     Allow user access to course modification.
 
-    `level` is one of ['instructor', 'staff', 'beta']
+    `level` is one of ['instructor', 'staff', 'beta', 'ccx_coach']
```

Comment:
Why did you make this change? I'm not sure what this permission does but we generally don't want coaches to modify the course. 

That said, beta users shouldn't be able to modify the course either...